utils/cmpcodesize/cmpcodesize/regex.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+
+# Cache the compiled regex into a global object.
+_ARCHITECTURE_REGEX = re.compile('architecture ([\S]+)')
```

Comment:
Indeed it can, thanks for pointing that out! Just changed it and the unit tests still all pass. feelsgoodman. :smile: